tests/unit/test__pandas_helpers.py

```diff
@@ -521,43 +518,6 @@ def test_bq_to_arrow_array_w_nullable_scalars(module_under_test, bq_type, rows):
     assert rows == roundtrip
 
 
-@pytest.mark.parametrize(
```

Comment:
Is it safe to remove this test completely? It still seems possible to use pandas 1.x